src/string_input_popup.cpp

```diff
@@ -490,6 +505,12 @@ const std::string &string_input_popup::query_string( const bool loop, const bool
             if( _position < static_cast<int>( ret.size() ) ) {
                 ret.erase( _position, 1 );
             }
+        } else if( action == "HELP_KEYBINDINGS" ) {
+            ctxt->display_menu();
```

Comment:
This is handled by `input_context` automatically and doesn't need to be called here. In fact I think it will cause the keybindings menu to be opened twice.

Reply:
You are correct.  I added this before realizing that `F1` worked.  This will be removed.